frontend/config/webpack.prod.babel.js

```diff
@@ -0,0 +1,38 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
```

Comment:
note that you could have a single webpack config file as well, and just configure the JSON config object differently depending either on `NODE_ENV` or on `npm_lifecycle_event`. Like here https://github.com/belongapp/belong-website/blob/master/webpack.config.js